### PR TITLE
Add new locale for Bahasa Indonesia

### PR DESCRIPTION
**Pull request by yandearta**
## Summary by CodeRabbit

- **New Features**
	- Added Indonesian (Bahasa Indonesia) as a new language option in the translations management system and language switcher.
	- Introduced a new localization JSON file for Indonesian, providing comprehensive translations for various application sections.

- **Bug Fixes**
	- No bug fixes were included in this release.

- **Documentation**
	- No documentation updates were made in this release.

### CODE REVIEW

**Comment by NayamAmarshe** [comment]
@raniaamina Could you please review this PR? :D

File: renderer/locales/id.json [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    "TILE_SIZE_ERROR": {
+      "TITLE": "Kesalahan",
+      "DESCRIPTION": "Ukuran tile salah. Silakan ubah ukuran tile di pengaturan atau setel ke 0 ({data})"
+    },
+    "EXCEPTION_ERROR": {
+      "TITLE": "Kesalahan Sistem",
+      "DESCRIPTION": "Upscayl mengalami kesalahan. Mungkin, binari upscayl gagal mengeksekusi perintah dengan benar. Coba periksa log untuk melihat apakah Anda mendapatkan informasi. Anda dapat memposting masalah di repositori GitHub Upscayl untuk bantuan lebih lanjut."
+    },
+    "GENERIC_ERROR": {
+      "TITLE": "Kesalahan"
```
Comment (raniaamina) [inline renderer/locales/id.json:237]: Please use 'Galat' instead of 'Kesalahan' to all 'error' strings.

File: renderer/locales/id.json [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+          "NAME": "Upscayl Standar",
+          "DESCRIPTION": "Cocok untuk sebagian besar gambar."
+        },
+        "upscayl-lite-4x": {
+          "NAME": "Upscayl Lite",
+          "DESCRIPTION": "Cocok untuk sebagian besar gambar. Upscaling berkecepatan tinggi dengan kehilangan kualitas minimal."
+        },
+        "remacri-4x": {
+          "NAME": "Remacri (Non-Komersial)",
+          "DESCRIPTION": "Untuk gambar alami. "
```
Comment (raniaamina) [inline renderer/locales/id.json:134]: Better to use 'natural' for consistency with other.

File: renderer/locales/id.json [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    "LANGUAGE": {
+      "TITLE": "BAHASA UPSCAYL"
+    },
+    "CUSTOM_TILE_SIZE": {
+      "TITLE": "UKURAN TILE KUSTOM",
+      "DESCRIPTION": "Gunakan ukuran tile kustom untuk membagi gambar. Ini dapat membantu memproses gambar lebih cepat dengan mengurangi jumlah tile yang dihasilkan."
+    },
+    "TURN_OFF_NOTIFICATIONS": {
+      "TITLE": "MATIKAN NOTIFIKASI",
+      "DESCRIPTION": "Jika diaktifkan, Upscayl tidak akan mengirim pemberitahuan sistem apapun tentang keberhasilan atau kegagalan."
```
Comment (raniaamina) [inline renderer/locales/id.json:90]: Suggest: Jika diaktifkan, Upscayl tidak akan mengirim pemberitahuan sistem apapun ketika berhasil ataupun gagal.

File: renderer/locales/id.json [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+    },
+    "SUPPORT": {
+      "TITLE": "Mengalami masalah?",
+      "DOCS_BUTTON_TITLE": "🙏 MINTA BANTUAN",
+      "EMAIL_BUTTON_TITLE": "📧 EMAIL DEVELOPER"
```
Comment (raniaamina) [inline renderer/locales/id.json:95]: Suggest: SUREL PENGEMBANG

File: renderer/locales/id.json [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+      "DESCRIPTION": "Aktifkan Test Time Augmentation untuk hasil yang lebih baik, seperti menghilangkan artefak TAPI ini akan meningkatkan waktu pengolahan hingga 8x!"
+    },
+    "SYSTEM_INFO": {
+      "TITLE": "Info Sistem"
+    }
+  },
+  "APP": {
+    "TITLE": "Upscayl",
+    "BATCH_MODE": {
+      "TITLE": "Batch Upscayl",
```
Comment (raniaamina) [inline renderer/locales/id.json:108]: Suggest: Upscayl Massal

File: renderer/locales/id.json [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+      "TOTAL_BATCH_UPSCAYLS": "Total Batch Upscayl",
+      "TOTAL_IMAGE_UPSCAYLS": "Total Upscayl Gambar",
+      "TOTAL_DOUBLE_UPSCAYLS": "Total Double Upscayl",
+      "AVERAGE_UPSCAYL_TIME": "Waktu Rata-rata Upscayl",
+      "LAST_UPSCAYL_DURATION": "Durasi Upscayl Terakhir",
+      "LAST_USED_AT": "Terakhir Digunakan Pada"
+    },
+    "PROGRESS_BAR": {
+      "BATCH_UPSCAYL_IN_PROGRESS_TITLE": "Batch Upscayl Sedang Berlangsung:",
+      "IN_PROGRESS_TITLE": "Melakukan sihir Upscayl...",
```
Comment (raniaamina) [inline renderer/locales/id.json:185]: Sugpest: Upscayl Massal Sedang Berlangsung:

File: renderer/locales/id.json [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+      "STOP_BUTTON_TITLE": "STOP"
```
Comment (raniaamina) [inline renderer/locales/id.json:186]: Suggest: HENTIKAN

File: renderer/locales/id.json [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+      "CLOSE": "Tutup"
+    }
+  },
+  "ERRORS": {
+    "GPU_ERROR": {
+      "TITLE": "Kesalahan GPU",
+      "DESCRIPTION": "Mengalami masalah dengan GPU. Silakan baca dokumen untuk pemecahan masalah! ({data})"
+    },
+    "COPY_ERROR": {
+      "TITLE": "Kesalahan Salin",
```
Comment (raniaamina) [inline renderer/locales/id.json:221]: Wrong translation, it should: Salin Galat, this for copying an error log.

**Review by raniaamina — CHANGES_REQUESTED** [review]

**Comment by raniaamina** [comment]
> @raniaamina Could you please review this PR? 😄

Sure, done.

File: renderer/locales/id.json (hunk 1 above)
Comment (yandearta) [reply renderer/locales/id.json:237]: Thank you for the suggestion! While 'galat' is a valid term, it feels more formal than 'kesalahan,' which aligns better with the app's tone.

File: renderer/locales/id.json (hunk 2 above)
Comment (yandearta) [reply renderer/locales/id.json:134]: Thanks! Agreed, I'll update it for consistency.

File: renderer/locales/id.json (hunk 3 above)
Comment (yandearta) [reply renderer/locales/id.json:90]: Thanks! Agreed, I'll make the change.

File: renderer/locales/id.json (hunk 5 above)
Comment (yandearta) [reply renderer/locales/id.json:108]: Thanks! Agreed.

File: renderer/locales/id.json (hunk 4 above)
Comment (yandearta) [reply renderer/locales/id.json:95]: Thank you for the suggestion! While 'Surel Pengembang' is technically correct, 'Email Pengembang' is more commonly understood and aligns better with user familiarity.

File: renderer/locales/id.json (hunk 1 above)
Comment (raniaamina) [reply renderer/locales/id.json:237]: No worry, now 'galat' commonly used to refer 'error'.

File: renderer/locales/id.json (hunk 4 above)
Comment (raniaamina) [reply renderer/locales/id.json:95]: Aprroved :")

File: renderer/locales/id.json (hunk 1 above)
Comment (yandearta) [reply renderer/locales/id.json:237]: Thank you for the clarification! I'll update the strings to use 'galat' to ensure consistency and align with common usage.

**Review by raniaamina — APPROVED** [review]

**Pull request merged**